src/App.tsx

```diff
@@ -1,24 +1,87 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+
+import { Filter } from './types/FilterEnum';
 import { UserWarning } from './UserWarning';
+import { ErrorMessage } from './components/ErrorMessage';
+import { BottomPanel } from './components/BottomPanel';
+import { TodoList } from './components/TodoList';
+
+const USER_ID = 10268;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterOfTodo, setFilterOfTodo] = useState(Filter.ALL);
+  const [hasError, setHasError] = useState(false);
+
+  const changeFilterOfTodo = useCallback((status: Filter) => {
+    setFilterOfTodo(status);
+  }, []);
+
+  const closeErrorMessage = useCallback(() => {
+    setHasError(false);
+  }, []);
+
+  const filterTodos = useCallback(() => {
```

Comment:
Just instantly make a filtering
```suggestion
  const filteredTodos = useMemo(() => {
```